.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "tracecompass-test-traces"]
 	path = tracecompass-test-traces
 	url = https://git.eclipse.org/r/tracecompass/tracecompass-test-traces.git
+[submodule "org.eclipse.tracecompass.incubator"]
+	path = org.eclipse.tracecompass.incubator
+	url = https://git.eclipse.org/r/tracecompass.incubator/org.eclipse.tracecompass.incubator.git
```

Comment:
Question... and it's really a question here... should we make trace compass test traces a submodule of the incubator? The incubator could probably use these traces too.

Reply:
> Question... and it's really a question here... should we make trace compass test traces a submodule of the incubator? The incubator could probably use these traces too.

Good idea. Maybe once incubator starts using them? -Through one first commit that does that, i.e., adds the submodule and uses some of those files (in incubator). In general, I think it is beneficial to reuse test-traces as a submodule in such repos that need them, rather than potentially duplicating test traces files across. And eventually, test-traces may grow with files such as this xml analysis one used herein. But for now it should be enough to simply add incubator's in.

Reply:
@marco-miller If I understand correctly, the purpose of having the incubator code base as submodule is to be able to build the trace server for the tests, right? Could we instead make use of the nightly build of the trace server located at the link below?
https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/

Reply:
> @marco-miller If I understand correctly, the purpose of having the incubator code base as submodule is to be able to build the trace server for the tests, right?

No, not exactly. It is only for the test code to be able to use one of incubator's xml files as extension test data. The server is to be spawned manually by the user before running these tests, as described in the README.

> Could we instead make use of the nightly build of the trace server located at the link below?
> https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/

User running these tests can always use it yes. Or run a locally built server; at will then.

Reply:
Ok thanks for the clarification.